src/remote-config/remote-config-api.ts

```diff
@@ -54,6 +54,27 @@ export interface RemoteConfigCondition {
   tagColor?: TagColor;
 }
 
+/**
+ * Interface representing a Remote Config condition in the data-plane.
+ * A condition targets a specific group of users. A list of these conditions make up
+ * part of a Remote Config template.
+ */
```

Comment:
Let's get TW reviews on these. Thanks!

Reply:
Ah. Good point. I was focused on this being an internal client and overlooked that it updated the API. I've added @jenh as a reviewer.